src/MIDebugEngine/AD7.Impl/AD7StackFrame.cs

```diff
@@ -245,7 +245,11 @@ public void SetFrameInfo(enum_FRAMEINFO_FLAGS dwFieldSpec, out FRAMEINFO frameIn
 
         private void EnsureLocalsAndParameters()
         {
-            uint radix = Engine.CurrentRadix();
+            uint radix = _radix;
+            Engine.DebuggedProcess.WorkerThread.RunOperation(async () =>
```

Comment:
nit: This is kind of inefficient - we will switch to the worker thread and then immediately switch back to the main thread to obtain the display radix.

Suggestion: Split `CurrentRadix` into one function to obtain the Radix from AD7, and another to Set the radix. Something like:

Variable.cs: `await _engine.UpdateRadix(_engine.CurrentRadix());`
AD7StackFrame.cs:
```
    uint radix = Engine.CurrentRadix();
    if (radix != _debuggedProcess.MICommandFactory.Radix)
    {
       Engine.DebuggedProcess.WorkerThread.RunOperation(() => Engine.UpdateRadix(radix));
    });
```